scene/3d/light_3d.cpp

```diff
@@ -29,6 +29,14 @@
 /**************************************************************************/
 
 #include "core/config/project_settings.h"
```

Comment:
```suggestion
#include "light_3d.h"

#include "core/config/project_settings.h"
```
Though it's odd that this is included before the header in the first place